### DISCUSSION TITLE
Does the Student Pack Access to the Copilot come after the free trail?

### DISCUSSION THREAD

**Discussion by MikeTheMichaelS** [community/community#53194] [GitHub Education] [answered]
I got the student pack recently, and I am trying to access the copilot feature. However, when I open the copilot page in my settings (settings -> copilot), it prompt me for the free trail. Will the student pack access come after I finish my free trail?
Thanks

**Comment by samus-aran** [comment, marked as answer]
Hi @MikeTheMichaelS 

Yes, once the free trial session has ended you should be able to enable it as a verified student 👍
